browser/main/Detail/SnippetNoteDetail.js

```diff
@@ -91,6 +94,16 @@ class SnippetNoteDetail extends React.Component {
 
   componentWillUnmount () {
     if (this.saveQueue != null) this.saveNow()
+    ee.off('code:generate-toc', this.generateToc)
+  }
+
+  handleGenerateToc () {
+    const currentMode = this.state.note.snippets[this.state.snippetIndex].mode
+    if (currentMode.includes('Markdown')) {
+      const currentValue = this.refs['code-' + this.state.snippetIndex].value
+      const currentEditor = this.refs['code-' + this.state.snippetIndex].refs.code.editor
+      markdownToc.generate(currentValue, (modifiedValue) => { currentEditor.setValue(modifiedValue) })
```

Comment:
Can you remove the `{` and `}` to retain the code style. Like this:
```
(modifiedValue) => currentEditor.setValue(modifiedValue)
```

Reply:
Done 👍